docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -718,13 +718,112 @@ To read and write a value of the Bluetooth descriptor:
       - write about requestAttMtuChange()
  -->
 ### Accessing the ATT MTU of connected device
+
+To get the ATT MTU value or request change of the ATT MTU value:
+
+1. Get BLE adapter:
+   ```
+   var adapter = tizen.bluetooth.getLEAdapter();
+   ```
+2. Start scannining and search for a device:
+   ```
+   var device;
+   adapter.startScan(function onsuccess(scannedDevice) {
+   if (device.name == "MyBLEDevice") {
+      device = scannedDevice;
+      console.log("Found device: " + device.name + ", address: " + device.address);
+   }});
+   ```
+3. When the device will be found, stop scanning:
+   ```
+   adapter.stopScan();
+   ```
+4. Connect to the device:
+   ```
+   function onerror(e) {
+      console.log("Failed to connect to device: " + e.message);
+   }
+
+   function onconnected() {
+      console.log("Connected to device");
+   }
+
+   device.connect(onconnected, onerror);
+   ```
+5. If the device has been connected call `getAttMtu()` function on the device object to get current ATT MTU value:
```

Comment:
I think, that points 1-4 are redundant - the guide (https://docs.tizen.org/application/web/guides/connectivity/bluetooth/#connecting-to-a-bluetooth-low-energy-device) already has a manual, instructing the readers how to establish a connection with a GATT server (https://docs.tizen.org/application/web/guides/connectivity/bluetooth/#connecting-to-a-bluetooth-low-energy-device).

Please, take a look at other sections of the existing manual, e.g. https://docs.tizen.org/application/web/guides/connectivity/bluetooth/#receiving-notifications-on-characteristic-value-changes.
They don't list all the steps required to make a connection, but link to the proper, existing section of the manual:

> 1. [Connect to a Bluetooth Low Energy device.](https://docs.tizen.org/application/web/guides/connectivity/bluetooth/#connecting-to-a-bluetooth-low-energy-device)

Please, consider removing the points 1-4 from this section.

Reply:
Done